tests/components/group/test_sensor.py

```diff
@@ -266,24 +308,25 @@ async def test_sensor_incorrect_state(
 
     entity_ids = config["sensor"]["entities"]
 
+    # Check that the final sensor value is unavailable if a non numeric input exists
     for entity_id, value in dict(zip(entity_ids, VALUES_ERROR)).items():
         hass.states.async_set(entity_id, value)
         await hass.async_block_till_done()
 
     state = hass.states.get("sensor.test_failure")
-
-    assert state.state == "15.3"
+    assert state.state == "unknown"
     assert (
         "Unable to use state. Only numerical states are supported, entity sensor.test_2 with value string excluded from calculation"
```

Comment:
```suggestion
        "Unable to use state. Only numerical states are supported"
```
Let's use same as previous test